tyk-headless/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
-appVersion: "1.0"
 name: tyk-headless
+appVersion: "v3.2.1"
```

Comment:
IIRC, we've decided to remove `appVersion` altogether since it can't track all the components.

Reply:
I don't about it.  where was this decision published? regardless atm it's "1.0" and it's wrong. continue in slack

Reply:
I vaguely remember that it was made during our planning session. Maybe @excieve can provide more input on it?

As mentioned on slack, I'm inclined to remove appVersion in the Chart.yaml for all charts.